src/components/heading/heading.spec.js

```diff
@@ -92,6 +94,39 @@ describe("Heading", () => {
     expect(wrapper.find(StyledSubHeader).text()).toEqual("subheader");
   });
 
+  it("renders passed components inside pills container when provided", () => {
+    const testPills = [
+      <Pill key="1">test pill 1</Pill>,
+      <Pill key="2">test pill 2</Pill>,
+    ];
+    const wrapper = shallow(<Heading title="test" pills={testPills} />);
+    expect(wrapper.find(StyledHeadingPills).children().length).toEqual(2);
+    expect(wrapper.find(Pill).children().length).toEqual(2);
```

Comment:
is there a reason that you're checking pill children length? couldn't it just be `expect(wrapper.find(Pill)).toHaveLength(2);`?

Reply:
I want to make sure that additional container is rendered as has exact children as passed.

Reply:
I was asking why are you checking the children of Pill instead of Pill directly.
As you mentioned the container - it could also be checked using `toHaveLength`:
```
expect(wrapper.find(StyledHeadingPills).children()).toHaveLength(2);
expect(wrapper.find(Pill)).toHaveLength(2);
```